app/prism/Recommendations.scala

```diff
@@ -65,21 +68,20 @@ object Recommendations {
     }) getOrElse false
   }
 
-  def isObsoleteUbuntu(ubuntuAmi: AMI): Boolean = {
-    val DistExtractor = ".*/ubuntu-(\\w+?)-(\\d{2}.\\d{2})-.*".r
+  def isObsoleteUbuntu(ubuntuAmi: AMI, now: DateTime): Boolean = {
+    val DistExtractor = ".*/ubuntu-(\\w+?)-(\\d{2}).(\\d{2})-.*".r
```

Comment:
Is there a way to do this without using regex? or maybe we could add a comment explaining what kind of pattern this is looking for